R/backend-snowflake.R

```diff
@@ -18,13 +18,198 @@ NULL
 
 #' @export
 sql_translation.Snowflake <- function(con) {
-    sql_variant(
-      sql_translator(
-        .parent = base_odbc_scalar,
-        log10  = function(x) sql_expr(log(10, !!x))
-        ),
-      base_agg,
-      base_win
+  sql_variant(
+    sql_translator(
+      .parent = base_odbc_scalar,
+      log10 = function(x) sql_expr(log(10, !!x)),
+      grepl = snowflake_grepl,
+      round = snowflake_round,
+      paste = snowflake_paste(" "),
+      paste0 = snowflake_paste(""),
+      str_c = function(..., sep = "", collapse = NULL) {
+        if (!is.null(collapse)) {
+          abort(c(
```

Comment:
For consistency this should use `cli_abort()` (with `{.arg collapse}`, `{.fn str_c}`, ...)